src/Conf.ml

```diff
@@ -660,8 +660,8 @@ let inplace =
 let inputs =
   let docv = "SRC" in
   let doc =
-    "Input files. At least one is required, and exactly one without \
-     --inplace."
+    "Input files. If none are passed, will read from stdin. Exactly one is \
+     needed for --inplace."
```

Comment:
Question regarding user interface: would it be better to accept `-` as a single filename denoting `stdin` instead of an empty list of input files? I don't really know, but both approach seem to be common.